uisdk/src/main/res/values-nl/strings.xml

```diff
@@ -186,6 +186,8 @@
     <string name="kh_uisdk_K4014">Kon reis niet boeken als agent</string>
     <string name="kh_uisdk_K4015">Kon reis niet boeken als reiziger</string>
     <string name="kh_uisdk_K4018">Kon niet boeken omdat de offerte niet langer beschikbaar is</string>
+    <string name="kh_uisdk_K4020">Could not book trip with the selected DMS</string>
+    <string name="kh_uisdk_K4025">Quote price increased whilst booking.</string>
```

Comment:
We shouldn't add translations in en to foreign languages, because we'll just end creating bug tickets for missing translations in the future.